shims/common/src/main/java/org/apache/hadoop/hive/shims/Utils.java

```diff
@@ -203,100 +187,5 @@ public AppConfigurationEntry[] getAppConfigurationEntry(String appName) {
     }
   }
 
-  public static final String XSRF_CUSTOM_HEADER_PARAM = "custom-header";
-  public static final String XSRF_CUSTOM_METHODS_TO_IGNORE_PARAM = "methods-to-ignore";
-  private static final String XSRF_HEADER_DEFAULT = "X-XSRF-HEADER";
-  private static final Set<String> XSRF_METHODS_TO_IGNORE_DEFAULT = new HashSet<String>(Arrays.asList("GET", "OPTIONS", "HEAD", "TRACE"));
-
-  /*
-   * Return Hadoop-native RestCsrfPreventionFilter if it is available.
-   * Otherwise, construct our own copy of its logic.
-   */
-  public static Filter getXSRFFilter() {
```

Comment:
I'm not sure if the Hive community treats it as a public API, if there are concerns about compatibility, another approach is:

1. leaving `Utils` as-is, 
2. creating a `HadoopUtils`, copying all other methods to `HadoopUtils`
3. changing callers to use `HadoopUtils` instead of `Utils` to avoid initializing `Utils`

Reply:
Sounds like a more compatible solution.

Reply:
Either works for me. Looks like this method is only in one place.